python/plugins/grassprovider/description/r.path.txt

```diff
@@ -0,0 +1,13 @@
+r.path
+Traces paths from starting points following input directions.
+Raster (r.*)
+QgsProcessingParameterRasterLayer|input|Name of input direction
+QgsProcessingParameterEnum|format|Format of the input direction map|auto;degree;45degree;bitmask|false|0|false
+QgsProcessingParameterRasterLayer|values|Name of input raster values to be used for output|None|True
+QgsProcessingParameterRasterDestination|raster_path|Name for output raster path map
+QgsProcessingParameterVectorDestination|vector_path|Name for output vector path map
+QgsProcessingParameterPoint|start_coordinates|Map coordinates of starting point(s) (E,N)|None|True
```

Comment:
Looking at the GRASS docs it seems that this parameter can accept multiple points, so it uses plural for "coordinates". Maybe it is better to change parameter description to reflect that in Processing tool only single point is accepted?

Reply:
> Looking at the GRASS docs it seems that this parameter can accept multiple points, so it uses plural for "coordinates". Maybe it is better to change parameter description to reflect that in Processing tool only single point is accepted?

done